src/displaycontroller.h

```diff
@@ -1092,10 +1106,7 @@ class GenericBitmappedDisplayController : public BitmappedDisplayController {
         return;
       X1 = iclamp(X1, paintState().absClippingRect.X1, paintState().absClippingRect.X2);
       X2 = iclamp(X2, paintState().absClippingRect.X1, paintState().absClippingRect.X2);
-      if (paintState().paintOptions.NOT)
```

Comment:
genericAbsDrawLine no longer needs special-case code for handing `NOT` - instead the `rawFillRow` and `rawSetPixel` functions that get passed into it already understand how to handle the various painting modes